build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ -z $1 ]; then
```

Comment:
Use [[..]] instead of [..]. See https://google.github.io/styleguide/shellguide.html#test----and--- for rationale. In addition, enclose $1 in quotes. See https://google.github.io/styleguide/shellguide.html#quoting guidelines.